src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -282,7 +295,7 @@ protected Object decode(
                                     break;
                                 case 44:
                                     value = buf.readUnsignedShortLE();
-                                    position.set(Position.KEY_FUEL_LEVEL, (value < 65500) ? value : null);
+                                    position.set("fuel", (value < 65500) ? value : null);
```

Comment:
Why are we changing this?

Reply:
It's been mixed with conventional fuel gauges before...

name like field 44

Reply:
It is more correct to consider it just a fuel sensor without a number - it is only one. After it, fuel1 ... fuel16, we can have sensors from RS-485.
And separately it is necessary to select fuel from the CAN bus, I called it obdFuel - it is also the same, but with a variant in liters and percentages.

Reply:
But `Position.KEY_FUEL_LEVEL` is the same key, so this makes no sense.

Reply:
Yes, I saw it, but I thought that just "fuel" looks a little shorter.  Change to key?

Reply:
Yes. And can we please avoid any more changes in this PR, so we can finally merge it.

Reply:
Ok!